velox/exec/HashTable.cpp

```diff
@@ -998,10 +1007,10 @@ FOLLY_ALWAYS_INLINE void HashTable<ignoreNullKeys>::buildFullProbe(
     uint64_t hash,
     char* inserted,
     bool extraCheck,
-    int32_t partitionBegin,
-    int32_t partitionEnd,
+    PartitionBoundIndexType partitionBegin,
+    PartitionBoundIndexType partitionEnd,
     std::vector<char*>* FOLLY_NULLABLE overflows) {
-  auto insertFn = [&](int32_t /*row*/, int32_t index) {
+  auto insertFn = [&](int32_t /*row*/, PartitionBoundIndexType index) {
```

Comment:
If `PartitionBoundIndexType` is `int64_t`, will `setRowPointer` (L1018) work well?